gst/nnstreamer/tensor_query/nnstreamer_edge_common.c

```diff
@@ -37,8 +37,12 @@ nns_edge_event_create (nns_edge_event_e event, nns_edge_event_h * event_h)
   }
 
   memset (ee, 0, sizeof (nns_edge_event_s));
+  nns_edge_lock_init (ee);
+
+  nns_edge_lock (ee);
   ee->magic = NNS_EDGE_MAGIC;
   ee->event = event;
+  nns_edge_unlock (ee);
```

Comment:
lock of LINE 42-45 are meaningless. The object, "ee", cannot exist in another context.